### ISSUE TITLE
can I use it when some of my commits is not standard type

### ISSUE THREAD

**Issue by Marckon** [conventional-changelog/conventional-changelog#707]
most of my commits contain header, body, etc. But I run `npm version x.x.x`, and got commits like `x.x.x`, it's not standard thus I cannot generate changelog with cli.

**Comment by Wenting-Luo** [comment]
Have you found out how to include commits that do not have standard header?

**Comment by dangreen** [comment]
Non-conventional commits are tolerated; presets simply do not render them. Normal tags still delimit releases, and a custom parser/writer can include such commits if needed. Closing as completed.
